packages/mobile/locales/es-419/global.json

```diff
@@ -113,5 +113,6 @@
   "timeframes": {
     "30d": "30 dias"
   },
-  "message": "Mensaje"
+  "message": "Mensaje",
+  "paymentLimitReached": "El límite diario es {{dailyLimit}} {{dailyRemaining}} restante"
```

Comment:
What is the process for translations usually? This is most likely wrong.

Reply:
There is spanish channel on  slack